dev-packages/cli/README.md

```diff
@@ -109,15 +109,29 @@ Default color and icon themes can be configured in `theia.frontend.config` secti
   },
 ```
 
-### Build Target
+### Build Target & Target Config
 
 The following targets are supported: `browser` and `electron`. By default `browser` target is used.
 The target can be configured in the `package.json` via `theia/target` property, e.g:
 
 ```json
 {
     "theia": {
-        "target": "electron"
+        "target": "electron",
+        "frontend": {
+            "config": {
+                "electron": {
+                    "windowOptions": {
+                        "titleBarStyle": "hidden",
+                        "webPreferences": {
+                            "webSecurity": false,
+                            "nodeIntegration": true,
+                            "webviewTag": true
+                        }
+                    }
+                }
+            }
```

Comment:
I believe you can add a new section for the following so it is clearer:

https://github.com/eclipse-theia/theia/blob/33886b52c5a0efa4bd5d9035d9f0ef4c4654b6a9/dev-packages/application-package/src/application-props.ts#L131-L143

Here we are explicitly talking about the `target` field, not the `electron config`.